src/spryker/helm/app/templates/application/webapp/ingress.yaml

```diff
@@ -5,16 +5,16 @@ kind: Ingress
 metadata:
   creationTimestamp: null
   labels:
-    app.service: {{ .Values.resourcePrefix }}nginx
-  name: {{ .Values.resourcePrefix }}nginx
+    app.service: {{ .Values.resourcePrefix }}webapp
+  name: {{ .Values.resourcePrefix }}webapp
 spec:
   rules:
   {{- range $key, $value := .Values.docker.services.nginx.environment }}
   - host: {{ $value | quote }}
```

Comment:
FPM_HOST: localhost here will be weird. it shouldn't really use the env vars here anyway tho